MoreMenu.js

```diff
@@ -784,6 +784,25 @@
 				addListeners();
 			}
 			
+			if ($('.client-js .skin-vector #p-cactions').length) {
```

Comment:
maybe add a `&& typeof MutationObserver === "function"`

Reply:
Done in cbfc3a1